proto/enums.py

```diff
@@ -74,7 +74,7 @@ def __new__(mcls, name, bases, attrs):
                     descriptor_pb2.EnumValueDescriptorProto(name=name, number=number)
                     # Minor hack to get all the enum variants out.
                     for name, number in attrs.items()
-                    if isinstance(number, int)
+                    if name in attrs._member_names and isinstance(number, int)
```

Comment:
Can you explain why this check is needed?

Reply:
Fixed in https://github.com/googleapis/proto-plus-python/pull/492/commits/d72dfb431c6e529c234677c3d0fcf1ea6afc3404

`attrs.items()` contains more than just the members. `attrs._member_names` contains only the members

For example, see the result of `attrs.keys()`
```
attrs.keys()
dict_keys(['_generate_next_value_', '__module__', '__qualname__', '__firstlineno__', 'UNKNOWN', 'SQUAMOSA', 'DURASA', 'GIGAS', '__static_attributes__'])
```